src/workerd/api/pyodide/pyodide.c++

```diff
@@ -25,7 +27,11 @@ void PyodideBundleManager::setPyodideBundleData(
     kj::String version, kj::Array<unsigned char> data) const {
   auto wordArray = kj::arrayPtr(
       reinterpret_cast<const capnp::word*>(data.begin()), data.size() / sizeof(capnp::word));
-  auto messageReader = kj::heap<capnp::FlatArrayMessageReader>(wordArray).attach(kj::mv(data));
+  // We're going to reuse this in the ModuleRegistry for every Python isolate, so set the traversal
+  // limit to infinity or else eventually a new Python isolate will fail.
+  auto messageReader = kj::heap<capnp::FlatArrayMessageReader>(
+      wordArray, capnp::ReaderOptions{.traversalLimitInWords = UINT64_MAX})
```

Comment:
You can use `kj::maxValue` here which automatically becomes whatever the maximum is for the type being assigned to. Avoids mistakes if the type isn't what you expect. Also don't need to include `stdint.h`.

```suggestion
      wordArray, capnp::ReaderOptions{.traversalLimitInWords = kj::maxValue})
```